src/main/resources/styles/DarkTheme.css

```diff
@@ -0,0 +1,281 @@
+/* Dark theme for PeopleSoft */
+/* Inspired by app by behance user */
+/* Some colours from person link */
+
+.root {
+    /* JavaFX CSS Parser only supports
+       custom variables for colours and nothing else */
+    /* Source: https://stackoverflow.com/questions/13566210/declaring-variable-in-javafx-css-file */
+    /* Docs https://docs.oracle.com/javase/8/javafx/api/javafx/scene/doc-files/cssref.html */
+
+    /* Panel Colors */
+    base: white;
+    section-bg: base;
+    section-border: #ababab;
+    cmd-bg-and-box-outline: #cdcdcd;
+
+    tag: accent;
+    tag-alt: #FFAA02;
+
+    list-cell-even: #cdcdcd;
+    list-cell-odd: #dedede;
+    list-cell-selected: derive(list-cell-even, -10%);
+    selected: accent;
+
+    /* Font Colors */
+    text-default: black;
+    text-inverse: white;
+    text-error: #FF5736;
+    -fx-focus-color: #000000;
+
+    /* Final Colors */
+    bg: #1B1C31;
+    pane: #222338;
+    pane-light: #33344B;
+    accent: #6076FF;
+
+    text-norm: white;
+    text-faded: #858AAB;
+
+    tag-accent-bg: #282B4C;
+    tag-green: #19A589;
+    tag-green-bg: #19A589;
+    tag-yellow: #FF9860;
+    tag-yellow-bg: #382F3C;
+    tag-red: #E06767;
+    tag-red-bg: #352A3D;
+}
+
+.background {
+    -fx-background-color: bg;
+    background-color: bg; /* Used in the default.html file */
+}
+
+.table-view {
+    -fx-base: base;
+    -fx-control-inner-background: base;
+    -fx-background-color: base;
+    /*-fx-table-cell-border-color: transparent;*/
+    /*-fx-table-header-border-color: transparent;*/
+    -fx-padding: 5;
+}
+
+.table-view .column-header-background {
+    -fx-background-color: transparent;
+}
+
+.table-view .column-header, .table-view .filler {
+    -fx-size: 35;
+    -fx-border-width: 0 0 1 0;
+    -fx-background-color: transparent;
+    -fx-border-color:
+        transparent
+        transparent
+        transparent
+        transparent;
+    -fx-border-insets: 0 10 1 0;
+}
+
+.table-view .column-header .label {
+    -fx-font-size: 20pt;
+    -fx-font-family: "Segoe UI Light";
```

Comment:
is this font being used? is it available for macOS and Linux users as well (without installing additional fonts, or Microsoft apps, etc?)

Reply:
if it is, and it isn't available for non-Windows (or non-Microsoft-app users), then is it possible to use fallback fonts here?

Reply:
and all other uses of Segoe UI in this file, ofc

Reply:
inherited from AB-3, gonna change to Inter (google fonts) and provide the font files along with the build

Reply:
I have already included the files with this PR